cloudwatch_monitoring/positioning.py

```diff
@@ -5,16 +5,14 @@
 
 
 class Positioning:
```

Comment:
With this class as cut-down as it is, I question the need for it's existence. Perhaps we can just have some constants, dictionary, or tuple defined somewhere.

Reply:
Agreed, will adjust in next commit.

Reply:
done.